src/components/Result/Result.jsx

```diff
@@ -1,17 +1,40 @@
 import "./result.css";
 import { useNavigate } from "react-router-dom";
+import { useData } from "../../contexts";
+import { useState, useEffect } from "react";
 export const Result = () => {
+  const { data } = useData();
   const navigate = useNavigate();
+  const [score, setScore] = useState(0);
+  //const [p,setP]=useState(0)
+  // console.log("marked", data.markedAnswers);
+  // console.log("correct", data.correctAnswers);
```

Comment:
Remove commented code and console logging

Reply:
Ok, Thank you for pointing it out...